UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/ConfigurationFiles/NotificationTypes.json

```diff
@@ -0,0 +1,15 @@
+{
+    "EnsConfiguration": {
+
+        "NotificationTypes": [
+            {
+                "Name": "File Share Service - AVCS Data",
+                "TopicName": "fss-filesPublished"
+            },
+            {
+                "Name": "ADDS Data Pipeline",
+                "TopicName": "avcs-contentPublished"
+            }
+        ]
+    }
```

Comment:
Remove this duplicate file as NotificationTypes.json already added in API project.

Reply:
Removed this duplicate.